[error-tracking] Apply general doc update (#28724)
DataDog/documentation: content/en/error_tracking/explorer.md

    -   Graph of occurrences over time
    -   Number of occurrences in the selected time period

Issue are also tagged as:
- `New` if the issue was first seen less than two days ago and is in state **FOR REVIEW** (see [Issue States][5])
- `Regression` if the issue was **RESOLVED** and occurred again in a newer version (see [Regression Detection][6])
- `Crash` if the application crashed

> - Having a [Suspected Cause][3]

Reviewer: ```suggestion
- Having a [Suspected Cause][3]
```
Is this meant to be part of the `Crash` section?

Author: No these are different :)

{{< img src="/error_tracking/error-tracking-facets.png" alt="Click the pencil icon to hide or show available Error Tracking facets from view." style="width:100%;" >}}

### Issue level filters

In addition to error events, Error Tracking offers issue level filters to refine the list of displayed issues.

{{< img src="error_tracking/issue-level-filters.png" alt="Issue level filters in Error Tracking" style="width:100%;" >}}

#### Sources

Error Tracking consolidates errors from multiple Datadog products (Rum, Logs, APM) into a unified view, allowing you to watch and troubleshoot errors across your entire stack. You can choose to display **All**, **Browser**, **Mobile**, or **Backend** issues in the explorer.

> For more granular filtering, you can narrow down issues by specific log sources or SDK, per programming language.

Reviewer: ```suggestion
For more granular filtering, you can narrow down issues by specific log sources or SDK per programming language.
```
Does this change the intended meaning?

Author: Each log source has every programming language and each SDK has every programming language. Then you will know better than me how to phrase it haha

Reviewer: ```suggestion
For more granular filtering, you can narrow down issues by specific log sources or by SDK and scope to a programming language.
```
Something like this?
